dts/arm/st/u0/stm32u0.dtsi

```diff
@@ -273,6 +307,87 @@
 			status = "disabled";
 		};
 
+		dma1: dma@40020000 {
+			compatible = "st,stm32-dma-v2";
+			#dma-cells = <3>;
+			reg = <0x40020000 0x400>;
+			interrupts = <9 0 10 0 10 0 11 0 11 0 11 0 11 0>;
+			clocks = <&rcc STM32_CLOCK(AHB1, 0U)>;
+			dma-channels = <7>;
+			dma-offset = <0>;
+			dma-requests= <12>;
+			status = "disabled";
+		};
+
+		dmamux1: dmamux@40020800 {
+			compatible = "st,stm32-dmamux";
+			#dma-cells = <3>;
+			reg = <0x40020800 0x400>;
+			interrupts = <11 0>;
+			dma-channels = <12>;
```

Comment:
to be more compliant , STM32U073xx and STM32U083xx devices have 12 channels and STM32U031xx has 7 channels. 

We will have here  `dma-channels = <7>;`   and in stm32u073.dtsi  we will have at least 
```
dmamux1: dmamux@40020800 
{
	dma-channels = <12>;
	dma-requests = <76>;
};

```